R/deprecated.R

```diff
@@ -91,7 +85,7 @@ deprecate_soft <- function(when,
                            with = NULL,
                            details = NULL,
                            id = NULL,
-                           env = caller_env(2)) {
+                           env = caller_env(1)) {
```

Comment:
I think the `caller_env(2)` was correct? Since we want to check whether the caller (the user) of the caller (the developer) of `deprecate_soft()` inherits from global.

Reply:
Ooh, `env` is used for two purposes — to generate the message and to check where it's used. I think maybe those are two different environments.

Reply:
I'll back that change out of this PR.